### ISSUE TITLE
Accessibility fix - contrast on link colours

### ISSUE THREAD

**Issue by mrchrisadams** [climateaction-tech/branch-theme#12]
Some of our links are a tad low on contrast, which means we fail on some WCAG tests.

### How to replicate this.

You can open a browser in Firefox and run a check with the accessibility checker, but you can also use sitespeed's Axe checker.

Run this command with a recent version of nodejs (i.e. 12 or newer):

```
npx sitespeed.io https://branch.climateaction.tech/ --sustainable.enable --sustainable.pageViews 100000 --sustainable.useGreenWebHostingAPI --axe.enable -n 1 --headless
```

It'll generate a report, and if you navigate to the `pages/branch.climateaction.tech/1.html#axe`, you'll see the list of violations.

I've uploaded the generated zip file for convenience with the highlighted violations and the suggested fixes.

[2020-10-28-18-19-19.zip](https://github.com/climateaction-tech/branch-theme/files/5453752/2020-10-28-18-19-19.zip)
